cukinia

```diff
@@ -22,7 +22,60 @@ GENERIC_FUNCTS='cmd
 		test
 		user'
 
-# Take config file as first argument
+# Default log file set to stderr
+__log_file="&2"
```

Comment:
It looks weird to me logging to stderr. I'd rather have normal cukinia output (the test result) on it's standard output, and potential errors on stderr